### ISSUE TITLE
Formula sub-numbering and/or grouping

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/metanorma-itu#23]
In G.650.1, there are “sub-equations”, as you can see here. "6-16a” vs “6-16b” while normally they are like “6-17”. In Metanorma, we normally generate the equation numbers dynamically.

![image002](https://user-images.githubusercontent.com/11865/64980677-ebb68680-d8ec-11e9-965d-9d0a2eacdfac.png)

Is it absolutely necessary to use human-assigned sub-equation numbers, or is it alright to just label the equations individually?
 
In this case, the equation identifier “6-17” means:
“6” => Clause 6.
“17” => Sequence number of equation within Clause 6.
 
But “6-16a” means:
“6” => Clause 6.
“16” => Sequence number of equation within Clause 6.
“a” => Sequence of equation within…?

ITU:

> The use of the subnumbering means that (in this case) equations 6-16a and 6-16b are closely related, and that they can be referred to collectively as equation 6-16.
> This type of equation subnumbering is not very common within our Recommendations, but it could still be considered as a constraint of Recommendations developed using Metanorma. i.e. that equations need to be numbered sequentially.

If we do want to support sub-numbering, we can create a "formula group" which will group multiple formulas into the same sequence number. There are times where we need to group multiple equations.

**Comment by ronaldtse** [comment]
Discussed with @opoudjis , we may be able to handle this as:

1. Multiple Formula (`stem`/AsciiMath) constructs in source, grouped into a single block.
2. Each Formula gets rendered into MathML with numbering assigned by Metanorma.

The user is able to select the following options for an Equation Group:
* Numbering: turn on or off. If "on", the group is a single equation with sub-equation numbering (a, b, ...).
* Alignment: Whether the `=` signs within this group are aligned.

**Comment by opoudjis** [comment]
Ronald, we discussed something quite different to the scenario given in the excerpt above, so the solutions you are proposing are not applicable. 

We discussed multiline mathematical expressions in a single math block. The excerpt is discontinuous, and we would need to introduce a notion of subnumbering for discontinuous assets in general --- this is just as applicable to table 9a and table 9b, which may not be next to each other.

The solution will have to be generic to Metanorma, and it will be to introduce a `%subgroup` option. If that attribute is present on an asset, then it, and all immediately following assets also marked with the same subgroup attribute, will get an incremental letter suffix added to their label.
